material-stepper/src/main/res/layout/ms_step_tab.xml

```diff
@@ -27,6 +27,15 @@
             app:srcCompat="@drawable/ic_check"
             android:visibility="gone" />
 
+        <android.support.v7.widget.AppCompatImageView
```

Comment:
Hmmm you're about the styling. I guess it would be easier to have two separate views for that :)